src/css/variablesClassics.css

```diff
@@ -8,87 +8,32 @@ html {
   --a-display: block;
   --a-margin: 0 auto;
   --a-text-align: center;
-  --a-text-decoration: underline;
+  --a-text-decoration: none;
+  --orchestra-a-text-decoration: underline;
+  --tickets-a-text-decoration: underline;
+  --tickets-a-display: inline;
+  --a-text-decoration-hover: underline;
   --a-text-underline-offset: 0.2em;
   --background-color: #009fe3;
   --carousel-pagination-background-color-selected: white;
-  --carousel-pagination-background-color: var(--color);
+  --carousel-pagination-background-color: #FFED00;
```

Comment:
made some variables explicit so that they work on concerts-pages as well (where --color & --background-color switches)